CHANGELOG.rst

```diff
@@ -11,7 +11,16 @@ This document describes changes between each past release.
 - **BREAKING CHANGE** Use a hashed ``ADMIN_PASSWORD`` instead of a clear text one, ``./budget/manage.py generate_password_hash`` can be used to generate a proper password HASH (#236)
 - **BREAKING CHANGE** Turn the WSGI file into a python module, renamed from budget/ihatemoney.wsgi to budget/wsgi.py. Please update your Apache configuration!
 - Changed the recommended gunicorn configuration to use the wsgi module as an entrypoint
+- **BREAKING CHANGE** The default value of ``ADMIN_PASSWORD`` has changed. If you have a custom settings file which set ``ADMIN_PASSWORD`` to an empty string (""), the application will use the default admin password until you update your settings.
```

Comment:
I'm commenting here because it summarizes the intent: I'm not sure I like this behaviour. I'm saying this because having a default password in here seems like a security issue: instances with the default password will be easy targets.

One potential solution is to issue a warning and don't have any administration tasks available until a password is defined.